src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -453,8 +454,11 @@ private void populateOmopTable() throws Exception {
   @Test
   public void testSnapshotCreateFromRequest() throws Exception {
     populateOmopTable();
-    UUID snapshotRequestId = makeSnapshotAccessRequest().getId();
-    SnapshotSummaryModel snapshotSummaryByRequest = makeSnapshotFromRequest(snapshotRequestId);
+    SnapshotAccessRequestResponse approvedSnapshotAccessRequest =
+        approveSnapshotAccessRequest(makeSnapshotAccessRequest().getId());
```

Comment:
Can you stub the request so we don't have to approve it here? I feel like that could help make this test smaller. (No is totally okay, this is an integration test so its okay if it is a bit larger)

Reply:
Yeah, I could just modify the result of makeSnapshotAccessRequest to be approved instead of calling the approve endpoint